### ISSUE TITLE
시스템5 성능 실험 결과

### ISSUE THREAD

**Issue by bart2001** [nlp-research/biaffineparser#43]
시스템5 특징: 

- pre-trained word embedding: words.pos.original.vec
- pre-trained pos embedding: words.tag.original.vec
- n개의 워드 임베딩 평균 without dropout
- n개의 POS태그 임베딩 평균 without dropout
- 워드 평균 벡터 + POS태그 평균 벡터

 hyper parmeters
```
{
    "num_lstm_units": 400,
    "num_lstm_layers": 3,
    "arc_mlp_units": 500,
    "label_mlp_units": 100,
    "word_embed_size": 200,
    "pos_embed_size": 100,
    "optimizer": "adam",
    "learning_rate": 0.001,
    "decay_factor": 0.9,
    "num_train_epochs": 100,
    "train_filename": "data/sejong.train.csv",
    "dev_filename": "data/sejong.test.csv",
    "out_dir": "parser/model_dev_without_embed_dropout",
    "word_vocab_name": "word.pkl",
    "pos_vocab_name": "pos.pkl",
    "rel_vocab_name": "rel.pkl",
    "head_vocab_name": "head.pkl",
    "word_embed_file": "embeddings/words.pos.original.vec",
    "pos_embed_file": "embeddings/words.tag.original.vec",
    "embed_dropout": 0.0,
    "lstm_dropout": 0.33,
    "mlp_dropout": 0.33,
    "batch_size": 128,
    "device": "gpu",
    "debug": false
}
```
